src/Symfony/Bridge/Monolog/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.2.0
+-----
+ * The `$actionLevel` constructor argument of `Symfony\Bridge\Monolog\Handler\FingersCrossed\NotFoundActivationStrategy` has been deprecated and replaced by the `$inner` one which expects an ActivationStrategyInterface to decorate instead.
```

Comment:
Missing blank line above

Reply:
Done, I also mentioned that classes will become final in 6.0